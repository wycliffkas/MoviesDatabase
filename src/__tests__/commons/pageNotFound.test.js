import NotFound from "../../commons/NotFound";
import { shallow } from "enzyme";
import React from "react";

describe("Not Found Tests", () => {
  const wrapper = shallow(<NotFound />);
  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
