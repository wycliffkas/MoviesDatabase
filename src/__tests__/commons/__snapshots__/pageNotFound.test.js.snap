// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Tests should render correctly 1`] = `ShallowWrapper {}`;
